build/artifacts.props

```diff
@@ -215,7 +216,6 @@
     <PackageArtifact Include="Microsoft.EntityFrameworkCore.Sqlite.Core" Category="ship" AllMetapackage="true"/>
     <PackageArtifact Include="Microsoft.EntityFrameworkCore.SqlServer" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
     <PackageArtifact Include="Microsoft.EntityFrameworkCore.Tools" Category="ship" AppMetapackage="true" AllMetapackage="true" />
-    <PackageArtifact Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Category="ship" LZMA="true" PackageType="DotnetCliTool" />
```

Comment:
@ryanbrandenburg Do I need to revert this?

Reply:
Ah, I see, the problem wasn't that you removed the tool as a Package, it was that the reaction to that hadn't gotten through yet. You're other reaction's should be sufficient.